feeds/wifi-ax/hostapd/files/hostapd.sh

```diff
@@ -529,7 +529,7 @@ hostapd_set_bss_options() {
 	set_default signal_poll_time 5
 	set_default signal_drop_reason 3
 	set_default signal_strikes 3
-	set_default proxy_arp 0
+	set_default proxy_arp 1
```

Comment:
Should this behaviour be the same on WIFI5?

Reply:
wifi5 APs seem to forward broadcast ARP requests from one wireless client to another wireless client.  We do not need to enable proxy arp for this issue.  Enabling proxy arp in general by default as a feature for what it does is a good idea.  Perhaps can be done on the cloud side.